packages/legend-art/style/base/_common.scss

```diff
@@ -77,6 +77,17 @@
   margin: 0;
 }
 
+.dnd__entry__handle__container {
+  float: left;
+  cursor: move;
+  position: relative;
+  z-index: 1;
```

Comment:
can we comment why we need to set `position, z-index` and `--dragging { opacity: 0 }`?